pynicotine/gtkgui/ui/settings/downloads.ui

```diff
@@ -218,9 +219,10 @@
             </child>
             <child>
               <object class="GtkCheckButton" id="ShareDownloadDir">
-                <property name="label" translatable="yes">Share download folder</property>
+                <property name="label" translatable="yes">Share Download folder</property>
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
+                <property name="tooltip_text" translatable="yes">The equivalent of adding your Download folder as a public share.</property>
```

Comment:
Might be worth pointing out that downloaded files are added to your shares list automatically, not requiring a rescan.